index.html

```diff
@@ -242,27 +242,27 @@ <h3 class="title">
 <tr>
   <td>Rust</td>
   <td><a href="https://rs.delta.chat">rs.<wbr>delta.chat</a></td>
-  <td><a href="https://github.com/deltachat/deltachat-core-rust">deltachat/<wbr>deltachat-core-rust</a></td>
+  <td><a href="https://github.com/chatmail/core">chatmail/<wbr>core (formerly deltachat-core-rust)</a></td>
```

Comment:
```suggestion
  <td><a href="https://github.com/chatmail/core">chatmail/<wbr>core</a></td>
```

i do not think, this information is useful for ppl looking at this page. 

the page is mainly for discovering things, so, there is few need to think about that.

and if you're already discovered the "core", you will be redirected by some checkout and otherwise easily re-recognise.

it is also not that "deltachat-core-rust" is a brand that is known widely or was ever chosen carefully.

finally, this page is not updated often, so that "formerly" would tend to stay for years here :)

no blocker, however, i would not die on that hill :)